macsmc-battery/systemd/macsmc-battery-charge-control-end-threshold.path

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Monitor macsmc-battery's charge_control_end_threshold
```

Comment:
Does the quote here work properly? It sure seems to confuse GitHub's syntax highlighting, but I'm not sure what systemd does (couldn't find any other units that use quotes like this on my system).

Reply:
it works and `systemctl status macsmc-battery-charge-control-end-threshold.path` shows it corecctly
```
● macsmc-battery-charge-control-end-threshold.path - Monitor macsmc-battery's charge_control_end_threshold
     Loaded: loaded (/usr/lib/systemd/system/macsmc-battery-charge-control-end-threshold.path; static)
     Active: active (waiting) since Sat 2024-08-17 20:01:28 CEST; 3min 0s ago
   Triggers: ● macsmc-battery-charge-control-end-threshold.service

Aug 17 20:01:28 mbp13m2f systemd[1]: Started macsmc-battery-charge-control-end-threshold.path - Monitor macsmc-battery's charge_control_end_threshold.
```

I can remove it though since I agree that it looks a little off